libbeat/_meta/kibana/default/index-pattern/libbeat.json

```diff
@@ -12,5 +12,5 @@
       "version": 1
     }
   ],
-  "version": "6.1.1"
```

Comment:
Not related to this PR but I wonder why this index pattern is even checked in ...

Reply:
Solved in https://github.com/elastic/beats/pull/6069